crates/gitbutler-tauri/src/main.rs

```diff
@@ -37,7 +37,7 @@ fn main() {
     let mut tauri_context = generate_context!();
     gitbutler_secret::secret::set_application_namespace(&tauri_context.config().identifier);
 
-    let config_dir = dirs::config_dir()
+    let config_dir = but_path::app_config_dir()
         .expect("missing config dir")
         .join("gitbutler");
```

Comment:
I think this join should be skipped since the but_path::app_config_dir() adds the gitbutler folder.
@Byron

Reply:
I broke it :/, and how could I miss this :/.

Thanks for the quick fix, see #10379 for the follow-up.